src/main/io/rc_controls.c

```diff
@@ -302,6 +302,17 @@ void processRcStickPositions(rxConfig_t *rxConfig, throttleStatus_e throttleStat
 
 }
 
+bool isModeActivationConditionPresent(modeActivationCondition_t *modeActivationConditions, boxId_e modeId)
```

Comment:
This routine should be a replacement for the code in useRcControlsConfig() which checks if an arming switch is present (rewrite that callsite to use this function instead).